lib/openQAcoretest.pm

```diff
@@ -6,7 +6,7 @@ use testapi;
 sub get_log {
     my ($cmd, $name) = @_;
     my $ret = script_run "$cmd | tee $name";
-    upload_logs($name) if $ret;
+    upload_logs($name) if !$ret;
```

Comment:
this would upload if it succeeded - I guess the point was indeed the opposite

Reply:
No, it must upload the log file only if it has been created successfully.
I tested it, and previously, the file were created, but not uploaded.